src/Test.Utilities.Sarif/TestData.cs

```diff
@@ -54,6 +54,13 @@ public static SarifLog CreateOneIdThreeLocations()
                     new Run
                     {
                         Tool = CreateSimpleLog().Runs[0].Tool,
+                        VersionControlProvenance = new []
+                        {
+                            new VersionControlDetails
+                            {
+                                RepositoryUri = new Uri(@"https://bugfiler.visualstudio.com/")
```

Comment:
>https://bugfiler.visualstudio.com/ [](start = 58, length = 34)

use [https://bugfiler.example.com](https://bugfiler.example.com) for this URL. example.com is a reserved domain for illustrative purposes in docs (and code).

Reply:
I don't see why Nick can't use it. See [https://www.iana.org/domains/reserved](https://www.iana.org/domains/reserved). "Reserved" just means "not available for registration or transfer", _i.e._, nobody can own it.

---
In reply to: [397236588](https://github.com/microsoft/sarif-sdk/pull/1823#discussion_r397236588) [](ancestors = 397236588)

Reply:
Since this was an example, I had no strong preference.  Nice to know there's a domain dedicated to this.  Earlier versions used a visualstudio.com domain which might well be picked up.

---
In reply to: [398170991](https://github.com/microsoft/sarif-sdk/pull/1823#discussion_r398170991) [](ancestors = 398170991,397236588)